quantities/units/Si/Derived/Newton.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Quantities.Units.Si.Derived;
 
-public readonly struct Newton : ISiDerivedUnit, IForce
+public readonly struct Newton : ISiUnit, IForce
```

Comment:
IMHO the different namespaces are differentiation enough.
I try to use the names defined either by a governing body (SI) or by the consensus on wikipedia.
I should start to consistently add links to the respective Wikipedia article for every unit.
Also, I want to avoid having to "invent" new names, as there are way too many units for which we'd need to come up with naming policies. Namespaces are - as of now - the right way to resolve naming conflicts.

The other "Newton"'s namespace is IMHO distinct enough compared to this Newton's namespace:
`Quantities.Units.NonStandard.Temperature` vs `Quantities.Units.Si.Derived`, respectively.

Reply:
Done.